tests/drivers/wifi/nrf_wifi/testcase.yaml

```diff
@@ -11,24 +11,13 @@ tests:
     extra_configs:
       - CONFIG_NRF70_RADIO_TEST=y
       - CONFIG_NETWORKING=n
-      - CONFIG_WIFI_NM_WPA_SUPPLICANT=n
   nrf70.build.scan_only:
     extra_configs:
-      - CONFIG_NRF70_SCAN_ONLY=y
-      - CONFIG_NETWORKING=y
-      - CONFIG_NET_DHCPV4=n
-      - CONFIG_NET_NATIVE=n
-      - CONFIG_NET_OFFLOAD=y
-      - CONFIG_WIFI_NM_WPA_SUPPLICANT=n
+      - CONFIG_WIFI_USAGE_MODE_SCAN_ONLY=y
   nrf70.build.raw_scan_results:
     extra_configs:
+      - CONFIG_WIFI_USAGE_MODE_SCAN_ONLY=y
       - CONFIG_WIFI_MGMT_RAW_SCAN_RESULTS=y
-      - CONFIG_NRF70_SCAN_ONLY=y
-      - CONFIG_NETWORKING=y
-      - CONFIG_NET_DHCPV4=n
-      - CONFIG_NET_NATIVE=n
-      - CONFIG_NET_OFFLOAD=y
-      - CONFIG_WIFI_NM_WPA_SUPPLICANT=n
   nrf70.build.raw_tx:
```

Comment:
these modes need CONFIG_WIFI_NM_WPA_SUPPLICANT=y

Reply:
That has been enabled by default with the following in `drivers/wifi/nrf_wifi/Kconfig.nrfwifi`
```
configdefault WIFI_NM_WPA_SUPPLICANT_AP
   default y if WIFI_USAGE_MODE_AP || WIFI_USAGE_MODE_STA_AP
```